src/ephemeris/_config_models.py

```diff
@@ -10,6 +10,7 @@
 from pydantic import (
     BaseModel,
     Extra,
+    RootModel,
```

Comment:
Then we need to pin pydantic 2. I think the this would work with pydantic 1 if we are able to install a newer pydantic by bumping the python version.

Reply:
what about both, pydantic2 pin and py39 as minimal?

Reply:
yeah, let's do it. we've supported 2 for quite some time, just hope there's no other breakage or incompatibility.

Reply:
I forgot new lint will hop out with ne python 🤣